tests/tensorflow/test_models_zoo_tf.py

```diff
@@ -89,16 +96,12 @@ def test_trained_ocr_predictor(mock_tilted_payslip):
     out = predictor(doc)
 
     assert out.pages[0].blocks[0].lines[0].words[0].value == "Mr."
-    geometry_mr = np.array(
-        [[0.08844472, 0.35763523], [0.11625107, 0.34320644], [0.12588427, 0.35771032], [0.09807791, 0.37213911]]
-    )
-    assert np.allclose(np.array(out.pages[0].blocks[0].lines[0].words[0].geometry), geometry_mr)
+    geometry_mr = np.array([[0.1083984375, 0.0634765625], [0.1494140625, 0.0859375]])
```

Comment:
why is there fewer boxes ? :thinking:

Reply:
Some informations about this test:

1. It was not correct and should normally raise an Exception for example using `ocr_predictor` with another model as `db_resnet50_rotation` or `linknet_resnet18_rotation` and `assume_straight_pages=False` will raise an Exception because the other models are actually not trained on rotated samples but with the lower level predictors it still works because we check this case only in the ocr_predictor :sweat_smile:  (We will remove this limitation if all det models trained with rotated samples)
2. same for the coords `straighten_pages=True` and before it hasn't returned straight boxes which is not correct

Is this clear enough ? :sweat_smile:

Reply:
As you can see in the code changes before with `straighten_pages=True` we have rotated the image back and remapped the coords to the rotated image which makes in my mind no sense ... now you can also grab the corrected image directly from the predictor output (if you want) .. otherwise the garbage collector cleaned it (so after profiling with the main branch and this PR i have noticed exact 0 diff in mem usage)